gke-autopilot-hpa/Readme-update (#42)
GoogleCloudPlatform/click-to-deploy-solutions: gke-autopilot-hpa/README.md

[![banner](../banner.png)](https://cloud.google.com/?utm_source=github&utm_medium=referral&utm_campaign=GCP&utm_content=packages_repository_banner)

> # Build a High availability application easily with GKE autopilot

Reviewer: ```suggestion
# Build an auto scaling application easily with GKE autopilot
```
Hey @gruizmm HPA here stands for horizontal pod autoscaling.
